src/servers/main.cc

```diff
@@ -60,26 +60,44 @@ std::condition_variable exit_cv_;
 // exit if inference-server doesn't completely initialize (e.g. will
 // exit if even one model fails to load).
 bool exit_on_failed_init_ = true;
+constexpr size_t unique_service_endpoint_count_ = 4;
```

Comment:
This doesn't really make sense anymore. It could be the maximum number of unique endpoints... but I don;t think we need that (see other comments).

Reply:
It is for the four unique (or common) ports for status, health etc

Reply:
Yes, but given what we talked about yesterday they will not each (necessarily) have their own service. The common case (as it is today) is that all 4 will share the same service.

Reply:
Got it